.github/workflows/build_rust.yml

```diff
@@ -23,11 +23,15 @@ jobs:
             target: "aarch64-apple-darwin"
             lib-cache-key: turbo-lib-darwin-${{ inputs.release_branch }}
           - host: ubuntu-latest
+            container: ubuntu:xenial
+            container-options: "--platform=linux/amd64 --rm"
```

Comment:
I think we need to plumb these through to `container.options`: https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container#setting-container-resource-options

Reply:
🙏🏼 you're totally right, forgot to carry that forward from my test branch. Thank you